### PR TITLE
Wysiwyg images: refactor imageProps, parse, uploadDir

### LINKED ISSUE #1472: Wysiwyg should integrate with the Media Manager

### PR DESCRIPTION
**Pull request by kendallstrautman**
- Refactors the image props generation from `InlineWysiwyg` to `Wysiwyg`
- Adjusts `parse` to handle media object
- Replaces `directory` string with `uploadDir` function

**Todo**
---
- [x] merge master
- [x] update README if needed

### CODE REVIEW

**Comment by DirtyF** [comment]
The storybook is broken for me, did I need to rebuild the packages?

```
cd packages/react-tinacms-editor
npm run storybook
...
    Local:            http://localhost:9001/        │
```

Capture d’écran 2020-10-07 à 15 31 47

File: packages/react-tinacms-editor/src/components/Wysiwyg/index.tsx [hunk 1]
```diff
@@ -85,3 +98,36 @@ const Wrapper = styled.div`
   display: block;
   min-height: 100px;
 `
+
+function generateImageProps(
```
Comment (ncphillips) [inline packages/react-tinacms-editor/src/components/Wysiwyg/index.tsx:102]: What if this was `useImageProps` and the `useMemo` was called from in here?

Comment (kendallstrautman) [reply packages/react-tinacms-editor/src/components/Wysiwyg/index.tsx]: I like that!

**Comment by kendallstrautman** [comment]
This git history got a little hairy, i'm gonna open a new PR

**Pull request closed without merging**